lock-conjure-api/src/main/conjure/lock-api.yml

```diff
@@ -29,6 +40,12 @@ services:
     package: com.palantir.lock
     base-path: /lk
     endpoints:
+      lockAndGetHeldLocks:
+        http: POST /laghl/{namespace}
```

Comment:
missed an opportunity to have a `/laugh` endpoint

Reply:
I could rename the endpoint to `lockAndUltimatelyGetHeld` 😄